### ISSUE TITLE
Parse forms in request

### ISSUE THREAD

**Issue by hhurtado** [BBVA/kapow#49]
The GO's implementation doesn't allow to decode form encoded data sent in a request with an arbitrary method and/or content type other than "application/x-www-form-urlencoded". This is needed for Kapow! semantic so it MUST be changed in the future.

Implement a ParseForm function that doesn't care about method nor Content-Type.

**Comment by dumblob** [comment]
Yes, this is super important (at least for the use cases I had in mind).
